var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -312,6 +312,7 @@ def cmake_args(self):
                 '-DCMAKE_C_COMPILER=%s' % spec['mpi'].mpicc,
                 '-DCMAKE_CXX_COMPILER=%s' % spec['mpi'].mpicxx,
                 '-DCMAKE_Fortran_COMPILER=%s' % spec['mpi'].mpifc,
+                '-DMPIEXEC_EXECUTABLE=%s' % spec['mpi'].mpiexec
```

Comment:
@tamiko could you please confirm this is the right way to provide `mpiexec` to deal.II build system? I could not find much info in our docs...

@jppelteret FYI.